steve/runner.go

```diff
@@ -3,64 +3,81 @@ package steve
 import (
 	"bytes"
 	"context"
-	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"sync"
-	"sync/atomic"
 	"time"
 
 	"github.com/google/uuid"
 	"github.com/mailgun/holster/v4/collections"
 	"github.com/mailgun/holster/v4/syncutil"
+	"github.com/pkg/errors"
```

Comment:
This package is deprecated, please use `github.com/mailgun/holster/v4/errors` or built-in `errors`.